Server/src/genericInterfaces/AuthenticationInterfaces.ts

```diff
@@ -0,0 +1,19 @@
+export interface ILoginInformation {
+    email: string,
+    password: string
+}
+
+export interface ISignUpInformation {
+    email: string,
+    password: string,
```

Comment:
we arent gonna send a password directly though, backend will receive a passwordHash. 
and the hash would be as per this [article](https://security.blogoverflow.com/2013/09/about-secure-password-hashing/#:~:text=Passwords%20should%20be%20hashed%20with,providing%20most%20security%20is%20bcrypt.)
**Passwords should be hashed with either PBKDF2, bcrypt or scrypt**

Reply:
Its fine if the backend gets a password. Its hashed via argon then stored in the DB. The security concern here is if the database has a leak not if the request is intercepted

Reply:
ok i see, i read how it should work